main/src/ca/uwaterloo/flix/language/phase/unification/BoolUnification.scala

```diff
@@ -99,46 +83,22 @@ object BoolUnification {
 
     // Eliminate all variables.
     try {
-      val subst = successiveVariableElimination(query, freeVars)
-
-      //    if (!subst.isEmpty) {
-      //      val s = subst.toString
-      //      val len = s.length
-      //      if (len > 50) {
-      //        println(s.substring(0, Math.min(len, 300)))
-      //        println()
-      //      }
-      //    }
-
-      Some(subst)
+      Some(successiveVariableElimination(query, freeVars))
     } catch {
-      case ex: BoolUnificationException => None
+      case _: BoolUnificationException => None
     }
   }
 
   /**
-    * A heuristic used to determine the order in which to eliminate variable.
-    *
-    * Semantically the order of variables is immaterial. Changing the order may
-    * yield different unifiers, but they are all equivalent. However, changing
-    * the can lead to significant speed-ups / slow-downs.
-    *
-    * We make the following observation:
-    *
-    * We want to have synthetic variables (i.e. fresh variables introduced during
-    * type inference) expressed in terms of real variables (i.e. variables that
-    * actually occur in the source code). We can ensure this by eliminating the
-    * synthetic variables first.
-    */
-  private def computeVariableOrder(l: List[Int]): List[Int] = {
-    l.reverse // TODO have to reverse the order for regions to work
-  }
+   * Determine the variable order.
+   */
+  private def computeVariableOrder(l: List[Int]): List[Int] = l
```

Comment:
Do we need this?

Reply:
Not really, but I wanted to highlight there _is_ actually a variable order.